README.md

```diff
@@ -19,16 +19,27 @@ Use it:
 
 You'll get an XML file with your results in it.
 
-## More Permanent Usage
+## Permanent Usage
```

Comment:
It made more sense to me, command line usage, permanent usage.  Really, these could be under an H2 header `Usage`, then H3 `Command line` and H3 `Permanent`